modules/programs/autorandr.nix

```diff
@@ -79,6 +79,53 @@ let
         default = null;
         example = "left";
       };
+
+      transform = with types;
+        let
+          transformType = let
+            list3 = elemType: addCheck (listOf elemType) (l: length l == 3);
+          in uniq (list3 (list3 float));
```

Comment:
Why did you remove the `// { description = "3×3 matrix of floating point numbers"; };` part?